assets/app/scripts/filters/util.js

```diff
@@ -162,6 +162,15 @@ angular.module('openshiftConsole')
       return link;
     };
   })
+  .filter('dockerHubRepository', function() {
+    return function(repository) {
+      // Check if repository is in the form organization/repository
+      if (repository.match(/[a-zA-Z0-9-_.]+\/[a-zA-Z0-9-_.]/)) {
```

Comment:
I know this is a work in progress, but I noticed this regex will match garbage at the start and end of the string.

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/8552331/649c4626-24aa-11e5-8d02-b0833b9c2fed.png)

Nit: `String.search() !== -1` should be faster here than `String.match()`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/search